extensions/barrage/src/main/java/io/deephaven/extensions/barrage/chunk/SingleElementListHeaderReader.java

```diff
@@ -0,0 +1,74 @@
+//
```

Comment:
This doesn't get hit in the Java test, but presumably it is fine because we are exercising it with C++ tests on each nightly build?

Reply:
There are explicit unit tests for this; just not in the type-cross sectional use. (FlightMessageRoundTripTest)

Reply:
I ran full coverage with Stan's jacoco merge tool, and it is still a zero:
![image](https://github.com/user-attachments/assets/21243e45-c9b1-4c07-8655-842017506b1e)